tools/hedera-crosschain-bridge/scripts/deployments/deploy-hts-connector.ts

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import hre, { ethers } from 'hardhat';
+
+import { getNetworkConfigs, logExecutionSummary } from '../utils/helpers';
+
+async function main() {
+  const network = hre.network.name;
+  const [deployer] = await ethers.getSigners();
+
+  const networkConfigs = getNetworkConfigs(network);
+  if (!networkConfigs) throw new Error(`Network configuration not found for ${network}`);
+
+  const { blockExplorerUrl, lzEndpointAddress } = networkConfigs;
+
+  if (!lzEndpointAddress) {
+    throw new Error(`LayerZero endpoint not configured for ${network}`);
+  }
+
+  const tokenName = 'T_NAME';
```

Comment:
Should these be hardcoded instead of, for example, an env variable?

Reply:
Exported to `constants.ts`.